modules/ossm-tutorial-jaeger-generating-traces.adoc

```diff
@@ -28,31 +19,33 @@ This tutorial uses {ProductShortName} and the Bookinfo tutorial to demonstrate h
-.. Use the CLI to query for details of the route:
-+
-[NOTE]
-====
-Replace `<control_plane_project>` with the name of your control plane project. In this example, the control plane project is `istio-system`.
-====
+
+. After installing the Bookinfo sample application, send traffic to the mesh. Enter the following command several times.
 +
+[source,terminal]
 ----
-$ export JAEGER_URL=$(oc get route -n <control_plane_project> jaeger -o jsonpath='{.spec.host}')
+$ curl "http://$GATEWAY_URL/productpage"
 ----
 +
-. Launch a browser and navigate to `https://<JAEGER_URL>`.
-
-. If necessary, log in using the same user name and password as you use to access the {product-title} console.
+This command simulates a user visiting the `productpage` microservice of the application.
 
-. In the left pane of the Jaeger dashboard, from the Service menu, select "productpage" and click the *Find Traces* button at the bottom of the pane. A list of traces is displayed, as shown in the following image:
-
-+
-image::ossm-jaeger-main-screen.png[]
+. In the {product-title} console, navigate to *Networking* -> *Routes* and search for the Jaeger route, which is the URL listed under *Location*.
+* Alternatively, use the CLI to query for details of the route. In this example, `istio-system` is the control plane namespace:
 +
-. Click one of the traces in the list to open a detailed view of that trace.  If you click on the top (most recent) trace, you see the details that correspond to the latest refresh of the ``/productpage`.
+[source,terminal]
+----
+$ export JAEGER_URL=$(oc get route -n istio-system jaeger -o jsonpath='{.spec.host}')
+----
 +
-image::ossm-jaeger-spans.png[]
+.. Enter the following command to reveal the URL for the Jaeger console. Paste the result in a browser and navigate to that URL.
 +
-The trace in the previous figure consists of a few nested spans, each corresponding to a Bookinfo Service call, all performed in response to a ``/productpage` request. Overall processing time was 2.62s, with the *details* Service taking 3.56ms, the *reviews* Service taking 2.6s, and the *ratings* Service taking 5.32ms. Each of the calls to remote Services is represented by a client-side and server-side span. For example, the *details* client-side span is labeled `productpage details.myproject.svc.cluster.local:9080`. The span nested underneath it, labeled `details details.myproject.svc.cluster.local:9080`, corresponds to the server-side processing of the request. The trace also shows calls to *istio-policy*, which reflect authorization checks made by Istio.
+[source,terminal]
+----
+echo $JAEGER_URL
+----
+
+. Log in using the same user name and password as you use to access the {product-title} console.
+
+. In the left pane of the Jaeger dashboard, from the Service menu, select *productpage.bookinfo* and click the *Find Traces* button at the bottom of the pane. A list of traces is displayed.
```

Comment:
I think menus are enclosed in asterisks, like "*Service*".  I'm pointing it out in case your eyes skipped over it.